Problem 5:

 

Zingaro stone printing is a company that specializes in printing images or patterns on polished or unpolished stones. However, for the optimum level of printing of the image the stone surface has to have a Brinell's hardness index of at least 150. Recently, Zingaro has received a batch of polished and unpolished stones from its clients. Use the data provided to answer the following (assuming a 5% significance level);

 

5.1 Earlier experience of Zingaro with this particular client is favorable as the stone surface was found to be of adequate hardness. However, Zingaro has reason to believe now that the unpolished stones may not be suitable for printing. Do you think Zingaro is justified in thinking so?

5.2 Is the mean hardness of the polished and unpolished stones the same?

In [105]:
# import the important packages
import pandas as pd 
import numpy as np 
import matplotlib.pyplot as plt 
import seaborn as sns  
%matplotlib inline
import scipy.stats as stats  
from scipy.stats import norm
from scipy.stats import   ttest_1samp , ttest_ind
from statsmodels.formula.api import ols      # For n-way ANOVA
from statsmodels.stats.anova import _get_covariance,anova_lm # For n-way ANOVA
import statsmodels.stats.api as sm

In [134]:
d2=pd.read_csv('Zingaro_Company.csv')

In [135]:
d2.head()

Unpolished   Treated and Polished
0   164.481713            133.209393
1   154.307045            138.482771
2   129.861048            159.665201
3   159.096184            145.663528
4   135.256748            136.789227

In [136]:
d2.info();

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 75 entries, 0 to 74
Data columns (total 2 columns):
 #   Column                Non-Null Count  Dtype  
---  ------                --------------  -----  
 0   Unpolished            75 non-null     float64
 1   Treated and Polished  75 non-null     float64
dtypes: float64(2)
memory usage: 1.3 KB


Step 1: Define null and alternative hypotheses

H0=Mean Brinell’s hardness Index of unpolished stone surface is greater than or equal to 150. 

Ha= Mean Brinell’s hardness Index of unpolished stone surface is less than 150.

OR

𝐻0 :  𝜇  => 150

𝐻𝐴 :  𝜇  <  150



Step 2: Decide the significance level
Here we select 𝛼 = 0.05.

In [137]:
print("The sample size for this problem is",len(d2),'.')

The sample size for this problem is 75 .


Step 3: Identify the test statistic
We do not know the population standard deviation although the sample size is more than 30 still we use the t distribution and the  𝑡𝑆𝑇𝐴𝑇  test statistic.

Step 4: Calculate the p - value and test statistic

scipy.stats.ttest_1samp calculates the t test for the mean of one sample given the sample observations and the expected value in the null hypothesis. This function returns t statistic and the two-tailed p value. Here we will need answer in one tailed p value and that too 

In [138]:
# one sample t-test
# null hypothesis: expected value = 150
t_statistic, p_value = ttest_1samp(d2, 150)
print('One sample t test \nt statistic: {0} p value: {1} '.format(t_statistic, p_value))

One sample t test 
t statistic: [-4.1646296  -1.22891066] p value: [8.34257399e-05 2.22998968e-01] 


In [139]:
#p/2
8.34257399e-05/2

4.171286995e-05

So here for unpolished stones t statistic=-4.1 and p value for unploished stones is 4.171286995e-05

Step 5 Decide to reject or accept null hypothesis
Level of significance: 0.05

We reject the null hypothesis since p value < Level of significance

So the statistical decision is we reject the null hypothesis at 5% level of significance.

It means that there is sufficient evidence for Zingaro stone printing company to believe that unpolished stones are not suitable for printing, that is they have Brinell’s hardness index of less than 150. 

Answer 5.2

In [140]:
d2['Treated and Polished'].mean()

147.78811718133335

In [141]:
d2['Unpolished '].mean()

134.11052653373335

Mean for Unpolished is 134.11 and for Treated and Polished is 147.79. Therefore, the mean is not same